pages/common/exiv2.md

```diff
@@ -0,0 +1,28 @@
+# exiv2
+
+> Image metadata manipulation tool.
+> More information: <https://www.exiv2.org/manpage.html>.
+
+- Print a summary of the image Exif metadata:
+
+`exiv2 {{filename}}`
+
+- Print all metadata (Exif, IPTC, XMP) with interpreted values:
+
+`exiv2 -P kt {{filename}}`
+
+- Print all metadata with raw values:
+
+`exiv2 -P kv {{filename}}`
+
+- Delete all metadata from an image:
+
+`exiv2 -d a {{filename}}`
+
+- Delete all metadata, preserving the file timestamp:
+
+`exiv2 -d a -k {{filename}}`
+
+- Rename the file, prepending the date and time from metadata (not from the file timestamp):
+
+`exiv2 -r '%Y%m%d_%H%M%S_:basename:' {{filename}}`
```

Comment:
If the user can edit the value of the `-r` flag, perhaps it could go in `{{token_syntax}}`?
```suggestion
`exiv2 -r '{{%Y%m%d_%H%M%S_:basename:}}' {{filename}}`
```

Also, I notice that a number of these example have short arguments (e.g. `-d`, `-r`). In tldr pages, we generally prefer the long-form arguments (e.g. `--foo`), as it aids clarity. Do any of these arguments have long forms that we could use instead?

Reply:
Yes there're long options, but I thought the convention was to use short options if they were fairly self-explanatory / unambiguous from the description? Looking at the pages for grep, sort, ls, git-commit...